bin/propolis-server/src/lib/server.rs

```diff
@@ -448,6 +450,30 @@ async fn instance_ensure_common(
         }));
     }
 
+    let mut serial_task = server_context.services.serial_task.lock().await;
+    if serial_task.is_none() {
+        let (websocks_ch, websocks_recv) = mpsc::channel(1);
+        let (close_ch, close_recv) = oneshot::channel();
+
+        let serial = vm.com1().clone();
+        let err_log = rqctx.log.new(o!("component" => "serial task"));
+        let task = tokio::spawn(async move {
+            if let Err(e) = super::serial::instance_serial_task(
+                websocks_recv,
+                close_recv,
+                serial,
+                err_log.clone(),
+            )
+            .await
+            {
+                error!(err_log, "Spawning serial task failed: {}", e);
```

Comment:
extreme nit: I might drop the word "spawning" here; on my reading we've technically spawned the task but it failed for some other reason